packages/playwright-core/src/server/electron/electron.ts

```diff
@@ -80,6 +80,7 @@ export class ElectronApplication extends SdkObject {
       await electronHandle.evaluate(({ app }) => app.quit());
     });
     this._nodeSession.send('Runtime.enable', {}).catch(e => {});
+    this._nodeSession.send('Runtime.evaluate', { expression: '__playwright_run()' });
```

Comment:
Can you add a comment / description to the PR explaining why this is fixing the issue? I believe it will be very helpful for posterity.